### ISSUE TITLE
Extra USER arguments are not ignored even though dockerfilelint claims they are

### ISSUE THREAD

**Issue by rcjsuen** [replicatedhq/dockerfilelint#99]
```Dockerfile
FROM alpine
USER root node
```
```
$ docker run -v `pwd`/Dockerfile:/Dockerfile replicated/dockerfilelint /Dockerfile

File:   /Dockerfile
Issues: 2

Line 1: FROM alpine
Issue  Category      Title                 Description
    1  Clarity       Base Image Missing    Base images should specify a tag to use.
                     Tag

Line 2: USER root node
Issue  Category      Title                 Description
    2  Possible Bug  Extra Arguments       This command has extra arguments and will be ignored.
```
```
$ docker run -it test
docker: Error response from daemon: linux spec user: unable to find user root node: no matching entries in passwd file.
```
